packages/core/strapi/lib/services/entity-service/types/index.d.ts

```diff
@@ -10,7 +10,13 @@ export interface EntityService {
     uid: TContentTypeUID,
     params?: EntityService.Params.Pick<
       TContentTypeUID,
-      'fields' | 'filters' | 'pagination:offset' | 'sort' | 'populate' | 'publicationState' | 'plugin'
+      | 'fields'
+      | 'filters'
+      | 'pagination:offset'
```

Comment:
```suggestion
      | 'pagination'
```
I think [this supports both formats](https://docs.strapi.io/dev-docs/api/entity-service/order-pagination#pagination)

Reply:
I'm not sure about that, because here it doesn't appear: https://docs.strapi.io/dev-docs/api/entity-service/crud#findmany and actually, it doesn't work 😅 

with this code:
```
    const res = await strapi.entityService.findMany('api::test.test', {
      fields: ['name'],
      page: 1,
      pageSize: 1,
    });
    ctx.body = res;
```
I get this:
```
[
{
"id": 2,
"name": "test 1"
},
{
"id": 3,
"name": "test 2"
},
{
"id": 4,
"name": "test 3"
}
]
```

with this code 
```
    const res = await strapi.entityService.findMany('api::test.test', {
      fields: ['name'],
      limit: 1,
      start: 1,
    });
    ctx.body = res;
```
I get:
```
[
{
"id": 3,
"name": "test 2"
}
]
```

Reply:
Good to know, thanks!